### DISCUSSION TITLE
Do not have <custom> chart option

### DISCUSSION THREAD

**Discussion by qq20230530** [getredash/redash#6125] [Q&A] [answered]
Am using redash v10.
In my chart type dropdown, only have normal chart types like line, bar, scatter, etc..
According to official document, there should also be a <custom> type which allows users to customize their charts.
Have no idea why there is not such a chart type provided.
Does any one know the reason?
Do I need to install something extra???

**Comment by qq20230530** [comment]
https://github.com/getredash/redash/issues/5998
Asked the same question here, but did not get any reply.

**Reply by gaecoli** [reply]
Can you show some pics in your redash?

**Comment by gaecoli** [comment]
> #5998 Asked the same question here, but did not get any reply.



> Am using redash v10. In my chart type dropdown, only have normal chart types like line, bar, scatter, etc.. According to official document, there should also be a type which allows users to customize their charts. Have no idea why there is not such a chart type provided. Does any one know the reason? Do I need to install something extra???

`REDASH_FEATURE_ALLOW_CUSTOM_JS_VISUALIZATIONS` set this env in your .env file.

**Reply by gaecoli** [reply, marked as answer]
In your `.env` or `opt/env` file, set `REDASH_FEATURE_ALLOW_CUSTOM_JS_VISUALIZATIONS=true`.

**Reply by qq20230530** [reply]
Thank you.
Did what you mentioned above, but custom still does not show in the chart type list.

![image](https://github.com/getredash/redash/assets/135001174/47da7a50-dc15-4e5d-a63a-c8563219c434)

**Reply by qq20230530** [reply]
added this line in my env file:
![image](https://github.com/getredash/redash/assets/135001174/505b0684-53ec-4af4-9525-f110d4f18ca5)
restarted redash using commands below:
  400  sudo docker-compose down
  401  sudo docker-compose up -d
what else is needed?

**Reply by qq20230530** [reply]
my redash version:
![image](https://github.com/getredash/redash/assets/135001174/43596c18-22bd-44d0-948f-0ac9c63e4c27)

**Reply by qq20230530** [reply]
Custom showed up!
I think it takes redash some time to have this option shown.
Thanks a lot!!!

**Reply by gaecoli** [reply]
OK. I'm happy to help you.

**Comment by gaecoli** [comment]
<img width="744" alt="image" src="https://github.com/getredash/redash/assets/46274612/64a37829-8f2a-4a77-aa8f-e51fd3fd3a3e">
<img width="796" alt="image" src="https://github.com/getredash/redash/assets/46274612/b27c2dfc-ca94-4fa3-8d82-6eac3839cc05">


Default this env is `false`, so you must to set to `true`. And then `custom chart` will be showed.

**Reply by justinclift** [reply]
@gaecoli Should we change the default to `true` for this?

**Reply by gaecoli** [reply]
I think it's ok!

**Comment by gaecoli** [comment]
If you still have questions, you can contact me @qq20230530

**Reply by qq20230530** [reply]
I do not know much about plotly.
Now I have the custom option, is it possible to create radar chart using plotly on redash???

**Comment by gaecoli** [comment]
are you restart?

**Comment by gaecoli** [comment]
can you show me your docker-compose.yml?
